qiskit/circuit/quantumcircuit.py

```diff
@@ -1327,12 +1351,7 @@ def _substitute_parameters(self, parameter_map):
         parameter_map, substitute replacement for existing in all
         circuit instructions and the parameter table.
         """
-        for old_parameter, new_parameter in parameter_map.items():
-            for (instr, param_index) in self._parameter_table[old_parameter]:
-                new_param = instr.params[param_index].subs({old_parameter: new_parameter})
-                instr.params[param_index] = new_param
-                self._rebind_definition(instr, old_parameter, new_parameter)
-            self._parameter_table[new_parameter] = self._parameter_table.pop(old_parameter)
+        _ = self.bind_parameters(parameter_map, in_place=True)
```

Comment:
Can we remove `_substitute_parameters` since it's functionally replaced by `_substitute_parameter`?

Reply:
Yes!